scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -217,11 +221,11 @@ const processRelatedAssetIds = async (
 
   const relatedAssetsResult = await getRelatedAssetIds(assetId, assetData)
     .then(result => {
-      console.log(`Fetched related assets for ${assetId}`)
+      happyCount++
       return result
     })
-    .catch(e => {
-      console.error(`Error fetching related assets for ${assetId}: ${e}`)
```

Comment:
Errors are expected for some assets and actually quite useless - either we have it for some as expected, or for all and then granularity is useless just the same